### PR TITLE
Abdm improvements

### PR DESCRIPTION
**Pull request by khavinshankar**
## Proposed Changes

- add a refresh button in abdm facility records
- add a message to enable notification to verify patient in abdm fetch records modal

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

### CODE REVIEW

File: src/Components/ABDM/ABDMFacilityRecords.tsx [hunk 1]
```diff
@@ -21,8 +24,10 @@ const TableHeads = [
 ];
 
 export default function ABDMFacilityRecords({ facilityId }: IProps) {
+  const [key, setKey] = useState(0);
   const { data: consentsResult, loading } = useQuery(routes.abha.listConsents, {
```
Comment (rithviknishad) [inline src/Components/ABDM/ABDMFacilityRecords.tsx]:
```suggestion
  const { refetch, data: consentsResult, loading } = useQuery(routes.abha.listConsents, {
```

File: src/Components/ABDM/ABDMFacilityRecords.tsx [hunk 2]
```diff
@@ -21,8 +24,10 @@ const TableHeads = [
...
     query: { facility: facilityId, ordering: "-created_date" },
+    key: key.toString(),
```
Comment (rithviknishad) [inline src/Components/ABDM/ABDMFacilityRecords.tsx]:
```suggestion
```

File: src/Components/ABDM/ABDMFacilityRecords.tsx [hunk 3]
```diff
@@ -53,6 +58,15 @@ export default function ABDMFacilityRecords({ facilityId }: IProps) {
                         scope="col"
                         className="sticky right-0 top-0 py-3.5 pl-3 pr-4 sm:pr-6"
                       >
+                        <ButtonV2
+                          onClick={async () => {
+                            setKey((prev) => prev + 1);
+                          }}
```
Comment (rithviknishad) [inline src/Components/ABDM/ABDMFacilityRecords.tsx]:
```suggestion
                          onClick={() => refetch()}
```

**Review by rithviknishad — CHANGES_REQUESTED** [review]

**Review by rithviknishad — APPROVED** [review]

**Comment by nihal467** [comment]
change the warning message text to warning and re-use the warning icon from the file components

**Review by rithviknishad — APPROVED** [review]

**Comment by nihal467** [comment]
LGTM

**Comment by nihal467** [comment]
LGTM

**Review by gigincg — APPROVED** [review]

**Pull request merged**